pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -353,6 +350,7 @@ def __init__(  # pylint: disable=too-many-arguments
 
         # Creating the state vector
         self._mpi_handler = MPIHandler(mpi, mpi_buf_size, len(self.wires), c_dtype)
+        self._use_mpi = mpi
```

Comment:
```suggestion
        self._use_mpi = self._mpi_handler.use_mpi
```

Reply:
Would  `self._use_mpi = self._mpi_handler.use_mpi` be faster?

Reply:
Not faster but safer because we ensure that everything related to `MPI` is done by the handler :thinking: